WordPress/src/main/res/values/roles.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string-array name="roles" translatable="false">
+    <string-array name="roles" translatable="true">
         <item>
-            Administrator
+            Admin
```

Comment:
This introduces the discrepancy where the role name does not follow the same copy as Calypso. That is, Calypso uses "Administrator" for the name and only the abbreviated "Admin" for the role "badge".